test/extensions/http/cache/file_system_http_cache/cache_file_fixed_block_test.cc

```diff
@@ -76,7 +65,7 @@ TEST_F(CacheFileFixedBlockTest, ReturnsCorrectOffsets) {
 TEST_F(CacheFileFixedBlockTest, SerializesAndDeserializesCorrectly) {
   CacheFileFixedBlock block;
   block.setHeadersSize(100);
-  block.setBodySize(1000);
+  block.setBodySize(10000000000);
```

Comment:
nit: put a const above and express all large numbers in terms of that, like
```
constexpr uint64_t one_billion = 1000*1000*1000;
```
I feel like on some platform/compiler, 10000000000 would not compile because it's not got a size spec like ULL (non-portable) and won't fit in an int, but the above hack will work around that without excessive ugliness cause one billion will fit in an int.

Reply:
Good call. Absolutely bonkers that `uint64_t x = 10000000000` will compile and work [on clang] but `uint64_t x = 1000*1000*1000*10` does not.